substrate/bin/utils/chain-spec-builder/README.md

```diff
@@ -0,0 +1,117 @@
+# Chain Spec Builder
```

Comment:
For me, it makes little sense to copy the part of [other documentation]((https://github.com/paritytech/polkadot-sdk/blob/58fd5ae4ce883f42c360e3ad4a5df7d2258b42fe/substrate/bin/utils/chain-spec-builder/src/lib.rs#L19)) to the readme file. It would be better to move the everything to the readme file and then include it into the module documentation using:
```
#![doc = include_str!("../README.md")]
```

IMO copy is not maintainable.

Reply:
Also - we could just link existing doc page:
https://docs.rs/staging-chain-spec-builder/latest/staging_chain_spec_builder/
or polkadot-sdk doc (which is the same):
https://paritytech.github.io/polkadot-sdk/master/staging_chain_spec_builder/index.html

Reply:
Great suggestion! :+1:  I like better the first option because it should look the same for all three doc sources down the line, while maintaining a single file.